dbt/tests/data_tests/generic_tests/expect_date_frequency_ratio.sql

```diff
@@ -0,0 +1,52 @@
+{% test expect_date_frequency_ratio(
+    model,
+    compare_model,
+    multiplier,
+    date_column='report_date'
+) %}
+
+WITH model_years AS (
+    SELECT
+        EXTRACT(YEAR FROM {{ date_column }}) as year,
+        COUNT(DISTINCT {{ date_column }}) as date_count
+    FROM {{ model }}
+    WHERE {{ date_column }} IS NOT NULL
+    {% if 'data_maturity' in adapter.get_columns_in_relation(model) %}
+        AND (data_maturity != 'incremental_ytd' OR data_maturity IS NULL)
```

Comment:
Does it make sense to include `OR data_maturity IS NULL` here? If it were `NULL` wouldn't it also not equal `incremental_ytd`?